datafusion/sql/src/select.rs

```diff
@@ -297,6 +298,9 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         input: LogicalPlan,
         select_exprs: Vec<Expr>,
     ) -> Result<LogicalPlan> {
+        // Try process group by unnest
+        let input = self.try_process_aggregate_unnest(input)?;
```

Comment:
If unnest has already been processed by `try_process_aggregate_unnest`, does the following logic for handling `unnest` become redundant?

Reply:
I agree with you. We need to verify this logic in the future

Reply:
Tracked in https://github.com/apache/datafusion/issues/11498